Assets/HoloToolkit/Input/Scripts/Focus/IPointingSource.cs

```diff
@@ -11,12 +12,19 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public interface IPointingSource
     {
-        Ray Ray { get; }
+        bool InteractionEnabled { get; }
 
         float? ExtentOverride { get; }
 
+        [Obsolete]
+        Ray Ray { get; }
```

Comment:
Same as above, what replaces it?

Reply:
RayStep[] Rays { get; } replaces Ray Ray { get; }

See my initial post for a description of this pr's purpose and main changes.

Reply:
I think he's trying to suggest you add `[Obsolete("Please use RayStep[] Rays")]`

Reply:
Ah, got it. It sounded more like a general question 😆